carla_integration/drivers.launch

```diff
@@ -37,4 +37,10 @@ If not using simulated drivers they are activated if the respective mock argumen
   <arg name="mock_gnss"           value="$(eval 'gnss' in arg('mock_drivers').split())"/>
   <arg name="mock_lidar"          value="$(eval 'lidar' in arg('mock_drivers').split())"/>
 
+  <!-- Specific Drivers -->
+  <arg name="drivers" default="carla_ackermann_control_wrapper"
+    doc="Desired real drivers to launch specified by package name. Mock drivers will take precedence"/>
+
+  <arg name="ssc_interface_wrapper"         value="$(eval ('carla_ackermann_control_wrapper' in arg('drivers').split()) and not arg('mock_controller'))"/>
```

Comment:
I think in this use case you don't need all these conditionals here. You just need the <include> or <node> tag to launch your drivers.

Reply:
So in this case I think this whole file can just be 
```
<launch>
  <include file="$(find carla_ackermann_control_wrapper)/launch/calra_ackermann_control_wrapper.launch/>
  <include file="$(find calra_ackermann_control)/launch/carla_ackermann_control.launch/>
</launch>
```
But with comments/license etc.